### ISSUE TITLE
Meet Web Accessibility Guidelines

### ISSUE THREAD

**Issue by MJJBennett** [ufvcsa/ufvcsa.github.io#15]
[WCAG 2.1](https://www.w3.org/TR/WCAG21/) describes a set of guidelines that websites should follow in order to be as accessible as possible. Issues should be created for each subset of these guidelines (for example, Perceivable, Operable, etc) and, if necessary, subsets of those, and fixed where possible.

Because this will depend heavily on HTML content of the website, it should probably wait until after the Gatsby refactor (#12) as that will be changing the website layout quite a bit already, and may solve some of the accessibility concerns.

**Comment by MykalMachon** [comment]
As per #12 
@MJJBennett Should I make some changes to the markup/styling to help with this where possible in the Gatsby refactor? 

There are a number of things I can do[ from a semantic html standpoint](https://www.w3schools.com/html/html5_semantic_elements.asp) to make things more accessible in things like screen readers (using appropriate html tags such as `<header>` `<footer>` `<section>` `<article>` `<aside>`, etc)

**Comment by MykalMachon** [comment]
@MJJBennett 

**Good points!** I'll make a few changes where it's easy (specifically in the header and body) but it won't be much. Some of the work will actually be automated by Gatsby's Page Generator as it generates pages using up-to-date standards as far as semantic HTML. 

Thanks! I've been looking for an Open-Source project to contribute too for awhile! Considering this is a CSA/UFV specifc project I figured what better to work on than this! 

__If you're looking for some resources or have any questions about Gatsby don't hesitate to reach out over Discord!_
